modules/virt-deleting-virt-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/install/uninstalling-virt.adoc
+
+[id="virt-deleting-virt-cli_{context}"]
+= Deleting the {VirtProductName} using the CLI
+
+You can delete {VirtProductName} by using the CLI.
+
+.Prerequisites
+
+- `oc` command installed on workstation.
+
+- Access to an {product-title} cluster using an account with
+`cluster-admin` permissions.
+
+.Procedure
+
+. Delete the HyperConverged Custom Resource:
++
+----
+$ oc delete HyperConverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Delete the subscription:
```

Comment:
Which subscription?

Reply:
The subscription of the Openshift virtualization operator in Operator Lifecycle Manager (OLM).

Reply:
Updated the content.